os/arch/arm/src/Makefile

```diff
@@ -157,6 +160,25 @@ ifeq ($(BOARDMAKE),y)
   LDLIBS += -lboard
 endif
 
+ifeq ($(CONFIG_ARCH_BOARD_CY4390X),y)
```

Comment:
There is simple way to add pre-built library.
For example, on s5j chip
```
ifeq ($(CONFIG_S5J_SSS),y)
VPATH += chip/sss

# built-in static library
EXTRA_LIBS += chip/sss/libispdriver.a
endif
```

Basically we should put codes on approprietary location. This file is for whole part of arm architecture, not for specific part.
Please use above method instead of this and locate it on chip and board makefile.

Reply:
it will be changed as your guidance.

BR, HJ Lee.